expyfun/_eyelink_controller.py

```diff
@@ -547,7 +547,7 @@ def custom_calibration(self, ctype='HV5', horiz=2./3., vert=2./3.,
         --------
         EyelinkController.calibrate
         """
-        allowed_types = ['HV5']
+        allowed_types = ['HV5', 'HV9', 'H3', 'HV13']
```

Comment:
Order bugs me :-) Let's do H3, HV5, HV9, HV13.

Also, can't comment there, but please change docs to mention supported types.